app/ui-react/packages/atlasmap-adapter/src/DataMapperAdapter.tsx

```diff
@@ -83,9 +83,11 @@ export const DataMapperAdapter: React.FunctionComponent<IDataMapperAdapterProps>
       onMappingChange={onMappings}
     >
       <Atlasmap
-        showImportAtlasFileToolbarItem={false}
-        showExportAtlasFileToolbarItem={false}
-        showResetToolbarItem={false}
+        allowImport={false}
+        allowExport={false}
+        allowReset={false}
+        allowDelete={false}
```

Comment:
@igarashitm I took the liberty to update your PR with these 2 options required to fully disable features we don't want in Syndesis.

Reply:
thx!